auction-app-backend/src/test/java/com/example/auctionapp/service/ProductServiceTest.java

```diff
@@ -148,7 +149,7 @@ public void whenGetSortedProductsByName_thenReturnSortedProducts() {
     public void whenGetLatestProducts_thenReturnProductsSortedByStartDateDesc() {
         UUID categoryId = UUID.randomUUID();
 
-        GetProductRequest getProductRequest = new GetProductRequest(categoryId, "", "startDate", "DESC", 0, 10);
+        GetProductRequest getProductRequest = new GetProductRequest(categoryId, "", "startDate", "DESC", 0, 10, new ArrayList<>(), BigDecimal.ZERO, BigDecimal.valueOf(10000));
```

Comment:
make 10000 a constant
Also is this a good approach, to set max price if we don't know it 
I believe you've followed UI, but there max price i just a placeholder

Reply:
i fixed it into a constant now, but i'm a little confused about the question. dont we know the max price from the test? because we're using mocking here with some dummy data, we know what price we'll set for the data beforehand